net/postfix/src/opnsense/mvc/app/views/OPNsense/Postfix/general.volt

```diff
@@ -0,0 +1,106 @@
+            </div>
+            {{ partial("layout_partials/base_form",['fields':antispamForm,'id':'frm_antispam_settings'])}}
+            <hr />
+            <div class="col-md-12">
+                <button class="btn btn-primary"  id="saveAct2" type="button"><b>{{ lang._('Save') }}</b><i id="saveAct2_progress" class=""></i></button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script type="text/javascript">
+    $( document ).ready(function () {
+        var data_get_map = {'frm_general_settings':"/api/postfix/general/get"};
+        mapDataToFormUI(data_get_map).done(function (data) {
+            formatTokenizersUI();
+            $('.selectpicker').selectpicker('refresh');
+        });
+        var data_get_map2 = {'frm_antivirus_settings':"/api/postfix/antispam/get"};
+        mapDataToFormUI(data_get_map2).done(function(data){
+            formatTokenizersUI();
+            $('.selectpicker').selectpicker('refresh');
+        });
+        ajaxCall(url="/api/postfix/service/status", sendData={}, callback=function (data, status) {
+            updateServiceStatusUI(data['status']);
+        });
+
+	// check if Rspamd plugin is installed
+        ajaxCall(url="/api/postfix/service/checkrspamd", sendData={}, callback=function(data,status) {
+	    if (data == "0") {
+                $('#missing_rspamd').show();
+            }
+        });
+
+        // link save button to API set action
+        $("#saveAct").click(function () {
+            saveFormToEndpoint(url="/api/postfix/general/set", formid='frm_general_settings',callback_ok=function () {
+            $("#saveAct_progress").addClass("fa fa-spinner fa-pulse");
+                ajaxCall(url="/api/postfix/service/maketransport", sendData={});
+                ajaxCall(url="/api/postfix/service/reconfigure", sendData={}, callback=function (data,status) {
+                    ajaxCall(url="/api/postfix/service/status", sendData={}, callback=function (data,status) {
+                        updateServiceStatusUI(data['status']);
+                    });
+                    $("#saveAct_progress").removeClass("fa fa-spinner fa-pulse");
+                });
+            });
+        });
+        $("#saveAct2").click(function(){
+            saveFormToEndpoint(url="/api/postfix/antispam/set", formid='frm_antispam_settings',callback_ok=function(){
+		    $("#saveAct2_progress").addClass("fa fa-spinner fa-pulse");
+                ajaxCall(url="/api/postfix/service/maketransport", sendData={});
+                ajaxCall(url="/api/postfix/service/reconfigure", sendData={}, callback=function(data,status) {
+                    ajaxCall(url="/api/postfix/service/status", sendData={}, callback=function(data,status) {
+                        updateServiceStatusUI(data['status']);
+                    });
+	            $("#saveAct2_progress").removeClass("fa fa-spinner fa-pulse");
+                });
+            });
+        });
```

Comment:
remove the duplicate code

Reply:
How do I implement same button within two tabs?

Reply:
https://github.com/opnsense/plugins/blob/master/security/tor/src/opnsense/mvc/app/views/OPNsense/Tor/general.volt#L58-L61